Terminal.Gui/ConsoleDrivers/NetDriver/NetMainLoop.cs

```diff
@@ -16,7 +16,7 @@ internal class NetMainLoop : IMainLoopDriver
 
     private readonly ManualResetEventSlim _eventReady = new (false);
     private readonly CancellationTokenSource _inputHandlerTokenSource = new ();
-    private readonly ConcurrentQueue<NetEvents.InputResult?> _resultQueue = new ();
+    private readonly ConcurrentQueue<NetEvents.InputResult> _resultQueue = new ();
```

Comment:
Please, see first the #204. We may get rid of the `ManualResetEventSlim` in `NetEvents`.